test/test_respondent_controller.py

```diff
@@ -1215,6 +1215,30 @@ def test_put_email_verification_uses_case_insensitive_email_query(self):
             account_controller.put_email_verification(token)
             query.assert_called_once_with("test@example.test", db.session())
 
+    def test_email_verification_resets_password_reset(self):
+        self.populate_with_respondent()
+        db_respondent = respondents()[0]
+        token = self.generate_valid_token_from_email(db_respondent.email_address)
+        response = self.put_email_verification(token, 200)
+        db_respondent = respondents()[0]
```

Comment:
Not sure why this has been re-assigned to the same thing as above

Reply:
This updates the respondent variable which would otherwise fail the tests as it still contains the verification token and counter is 1